### PR TITLE
Satisfy Charmhub requirements

### PR DESCRIPTION
**Pull request by Gleland**
add charmcraft and metadata changes to build with charmcraft pack successfully

### CODE REVIEW

File: charmcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+          channel: "20.04"
+          architectures:
+              - amd64
+        - name: ubuntu
+          channel: "18.04"
+          architectures:
+              - amd64
+parts:
+  charm:
+    charm-python-packages: [setuptools < 58]
```
Comment (afreiberger) [inline charmcraft.yaml]: why is this necessary?  I don't see jinja2 or charmhelpers in requirements.txt

**Review by afreiberger — CHANGES_REQUESTED** [review]

File: charmcraft.yaml (hunk 1 above)
Comment (Gleland) [reply charmcraft.yaml]: thanks! i've updated this, and the list/indentation usage to be consistent as well.
